manim/mobject/matrix.py

```diff
@@ -64,6 +88,8 @@ def vector_coordinate_label(vector_mob, integer_labels=True, n_dim=2, color=WHIT
 
 
 class Matrix(VMobject):
+    """An mobject that displays matrix on the screen."""
```

Comment:
```suggestion
    """A mobject that displays a matrix on the screen."""
```

Reply:
Hi @kilacoda  Have incorporated your suggestion. "A" vs "An" ... the English-speaking world is still undecided on this :)  ... "An" is supposed to be used with "a,e,i,o,u" but there are exceptions. Nevertheless, I have incorporated your suggestion. I used "An" because I saw its usage in some other files as well.

Reply:
"An" is used before vowel sounds, "A" before consonant sounds. An apple, an hour; a banana, a unicorn.